src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java

```diff
@@ -1191,17 +1221,33 @@ public void testGetPaymentStatusTransition() throws Exception {
         HyperwalletStatusTransition returnValue;
         try {
             returnValue = client.getPaymentStatusTransition("pmt-2c059341-8281-4d30-a65d-a49d8e2a9b0f",
-                                                             "sts-1f7f58a9-22e8-4fef-8d6e-a17e2c71db33");
+                    "sts-1f7f58a9-22e8-4fef-8d6e-a17e2c71db33");
         } catch (Exception e) {
             mockServer.verify(parseRequest(functionality));
             throw e;
         }
+        List<HyperwalletLink> hyperwalletLinks = new ArrayList<>();
+        HyperwalletLink hyperwalletLink = new HyperwalletLink();
+        hyperwalletLink.setHref(
+                "https://uat-api.paylution.com/rest/v4/payments/pmt-2c059341-8281-4d30-a65d-a49d8e2a9b0f/status-transitions/sts-1f7f58a9-22e8-4fef"
+                        + "-8d6e-a17e2c71db33");
+        Map<String, String> mapParams = new HashMap<>();
+        mapParams.put("rel", "self");
+        hyperwalletLink.setParams(mapParams);
+        hyperwalletLinks.add(hyperwalletLink);
 
         assertThat(returnValue.getToken(), is(equalTo("sts-1f7f58a9-22e8-4fef-8d6e-a17e2c71db33")));
         assertThat(returnValue.getCreatedOn(), is(equalTo(dateFormat.parse("2017-11-16T02:04:17 UTC"))));
         assertThat(returnValue.getTransition(), is(equalTo(COMPLETED)));
         assertThat(returnValue.getFromStatus(), is(equalTo(CREATED)));
         assertThat(returnValue.getToStatus(), is(equalTo(COMPLETED)));
+        if (returnValue.getLinks() != null) {
+            HyperwalletLink actualHyperwalletLink = returnValue.getLinks().get(0);
+            HyperwalletLink expectedHyperwalletLink = returnValue.getLinks().get(0);
+            assertThat(actualHyperwalletLink.getHref(), is(equalTo(expectedHyperwalletLink.getHref())));
```

Comment:
This statement and the following two statements always return true as you are setting the value from returnValue.getLinks().get(0) to actualHyperwalletLink and expectedHyperwalletLink. Pls check the returnValue's link to that of hyperWalletLink

Reply:
Done.